cdap-app-templates/cdap-etl/cdap-etl-batch/pom.xml

```diff
@@ -63,6 +63,10 @@
       <artifactId>cdap-unit-test</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.hsqldb</groupId>
+      <artifactId>hsqldb</artifactId>
```

Comment:
I don't see this is added in the parent pom dependency management. Do I miss something?